site/docs/v1/tech/apis/applications.adoc

```diff
@@ -65,6 +66,72 @@ include::docs/v1/tech/apis/_standard-post-response-codes.adoc[]
 
 include::docs/v1/tech/apis/_application-response-body.adoc[]
 
+== Create a New Application From an Existing Application
+
+This API is used to create an Application from an existing Application.
```

Comment:
I would keep saying "new" to really drill that in and match the heading and key docs.

```suggestion
This API is used to create a new Application from an existing Application.
```

Reply:
I do like the heading to match, but I would just remove `new` from the heading.